bump-up.command

```diff
@@ -92,6 +92,23 @@ else
   echo "helix-rest/$ivy_file not exist"
 fi
 
+echo "bump up helix-lock/pom.xml"
+sed -i "s/${version}/${new_version}/g" helix-lock/pom.xml
+grep -C 1 "$new_version" helix-lock/pom.xml
+# git diff helix-lock/pom.xml
+
+ivy_file="helix-lock-"$version".ivy"
```

Comment:
Nit, I understand the other modules' names are also hardcoded. I would just make a variable `module_name="helix-lock"` (Or better, can read from pom file) and reuse the module. I've seen duplicated code in this script, which a for loop could handle. Maybe we can have another PR to clean this script.